opba-protocols/sandboxes/xs2a-sandbox/src/main/java/de/adorsys/opba/protocol/xs2a/testsandbox/internal/SandboxApp.java

```diff
@@ -286,7 +286,8 @@ private void doRunJar(StarterContext ctx) {
                             "--spring.config.location=" + buildSpringConfigLocation(ctx),
                             "--primary.profile=" + getPrimaryConfigFile(),
                             "--testcontainers.postgres.port=" + ctx.getDbPort().get(),
-                            "--logging.level.root=" + getSandboxLogLevel()
+                            "--logging.level.root=" + getSandboxLogLevel(),
+                            "--feign.client.config.default.loggerLevel=NONE"
```

Comment:
Instead of NONE, this should be implemented same way as getSandboxLogLevel() - to be configurable:
`"--feign.client.config.default.loggerLevel=" + getSandboxFeignLogLevel()`